OpenSim/Simulation/Model/PrescribedForce.cpp

```diff
@@ -42,14 +42,13 @@ using namespace std;
 /**
  * Default constructor.
  */
-PrescribedForce::PrescribedForce(Body* body)
+PrescribedForce::PrescribedForce(const PhysicalFrame* frame)
 {
```

Comment:
I agree it is cleaner to have default constructor that takes no arguments and a proper convenience constructor that  takes both the name of the `PrescribedForce` (and `const sted::string&`) and a `const` reference to a `PhysicalFrame`. I am OK with deprecating the constructor that takes a pointer OR removing it altogether.